.github/workflows/deploy.yml

```
@@ -0,0 +1,58 @@
+name: Deploy to prototype environment
+
+on:
+  workflow_run:
+    workflows: [ Run checks ]
+    types:
+      - completed
+    branches: [ main ] # Redundant, workflow_run events are only triggered on default branch (`main`)
+
+permissions:
+  contents: read
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+
+    steps:
+    - name: Install container dependencies
+      run: |
+        sudo apt-get update \
+        && sudo apt-get install -y --no-install-recommends \
+        libcurl4-openssl-dev
+
+    - uses: actions/checkout@v3
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.9"
+
+    - name: Install application dependencies
+      run: make bootstrap
+
+    - name: Deploy to cloud.gov
+      uses: 18f/cg-deploy-action@main
+      env:
+        DANGEROUS_SALT: ${{ secrets.DANGEROUS_SALT }}
+        SECRET_KEY: ${{ secrets.SECRET_KEY }}
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      with:
+        cf_username: ${{ secrets.cloudgov_username }}
+        cf_password: ${{ secrets.cloudgov_password }}
+        cf_org: gsa-10x-prototyping
+        cf_space: 10x-notifications
+        full_command: |
+          cf push --strategy rolling \
+          --var DANGEROUS_SALT="$DANGEROUS_SALT" \
+          --var SECRET_KEY="$SECRET_KEY" \
+          --var AWS_ACCESS_KEY_ID="$AWS_ACCESS_KEY_ID" \
+          --var AWS_SECRET_ACCESS_KEY="$AWS_SECRET_ACCESS_KEY"
```

[suggestion (nonblocking)] using this form was required for `api` because Christa uncovered a bug in the deployer that has now been fixed, so this whole thing can be simplified (a bit) to:

```suggestion
        push_arguments: >-
          --var DANGEROUS_SALT="$DANGEROUS_SALT"
          --var SECRET_KEY="$SECRET_KEY"
          --var AWS_ACCESS_KEY_ID="$AWS_ACCESS_KEY_ID"
          --var AWS_SECRET_ACCESS_KEY="$AWS_SECRET_ACCESS_KEY"
```

---

Do these vars need to be referenced here at all? Should we just set them directly via the cf cli instead of duplicating them in github secrets?

---

I'd rather leave them here and move towards a world where no human uses the `cf` command line at all.

---

Sounds good to me.